packages/sources/css/src/components/text-input/src/index.css

```diff
@@ -53,7 +53,7 @@ textarea.vtmn-text-input {
 }
 
 .vtmn-text-input::placeholder {
-  color: var(--vtmn-semantic-color_content-secondary);
+  color: var(--vtmn-semantic-color_content-tertiary);
```

Comment:
Why? It was an error?

Reply:
Yes, it resolves the issue refactor(@vtmn/css): new semantic colors #732.
JIRA ticket : https://decathlon.atlassian.net/jira/software/projects/DSRUN/boards/2922?selectedIssue=DSRUN-50

Reply:
Can we please do this separately with all the checklist listed here: https://github.com/Decathlon/vitamin-web/issues/732.
Thanks.